go/twofer: comments for disapprove (#1222)
exercism/website-copy: automated-comments/go/two-fer/work_with_any_provided_name.md

> This solution references the names `Alice` and `Bob` directly in the code. This is a good first step!

Reviewer: ```suggestion
Change the function to work with any name provided. Currently the solution only works with `Alice` and `Bob`, which is a first good step.
```

I would prefer to drop line 2, and have this be actionable.

Author: ```suggestion
```

> The funcion should be able to work with any name provided to it.

Reviewer 2: 'function' not 'funcion'.

We might make this hint a bit clearer. "This is a good first step, but one way to improve this function would be to make it handle any name provided to it, not just Alice and Bob specifically."

Reviewer: Instead of my suggestion above, I like this too. However, there is no "second" way to improve _this_ issue, so I would be it imperative.

```suggestion
This is a good first step. Improve this function to make it handle any name provided to it, not just Alice and Bob specifically.
```

Reviewer 2: To native English speakers, the imperative "Improve this..." strikes just _fractionally_ the wrong note. It sounds like a parent telling a child to wipe their face or tie their shoelaces. I like to try to set a collaborative tone, where the student and I explore the problem together. A tiny bit of extra politeness can really grease the wheels here.

Reviewer: That's probably true for _some_ native English speakers, but this is not a mentor doing collaboration here. It's a machine spitting out comments.  Personally this kind of politeness to me is actually _rude_. 

Alas, if you can make it actionable without sounding impolite, I'm super happy :heart:
